torch/_functorch/aot_autograd.py

```diff
@@ -1527,6 +1527,14 @@ def _detect_attribute_assignment(mod: torch.nn.Module):
         "_state_dict_hooks",
         "_state_dict_pre_hooks",
         "training",
+    ]
+    NN_MODULE_LAZY_STD_ATTRS = [
```

Comment:
Are they previously special handled in dynamo?

Reply:
This only applies to non-strict, since this bug was in detecting arribute assigments (in general Dynamo errors so we don't run this detection there).